src/commands/login.rs

```diff
@@ -95,6 +95,14 @@ pub struct LoginCommand {
     )]
     pub hippo_password: Option<String>,
 
+    /// Fermyon token
+    #[clap(
+        name = FERMYON_TOKEN,
+        long = "fermyon-token",
```

Comment:
It happens to be Fermyon-specific today, but I don't think the flag needs such a specific name.

```suggestion
        long = "token",
```

Reply:
done